canopy/app/src/images/grid.svg

```diff
@@ -0,0 +1,16 @@
+<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think the grid logo should be in here. The grid UI should supply it in the Grid repo.

Reply:
removed